docker/dockerfile

```diff
@@ -5,27 +5,32 @@ ARG USERID
 ARG USER
 
 # Setup environment
-ENV TERM linux
-ENV DEBIAN_FRONTEND noninteractive
+ENV TERM=linux
+ENV DEBIAN_FRONTEND=noninteractive
 RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 
 # Copy requirement files and install dependencies
 COPY docker/requirements.txt .
 RUN apt-get update && apt-get install --no-install-recommends -y $(cat requirements.txt)
 RUN rm requirements.txt
 
-# Install Gazebo Fortress
-RUN sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
-RUN wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
-RUN sudo apt-get update
-RUN sudo apt-get install ignition-fortress -y
+# Add the Gazebo repository and install Gazebo Harmonic
+RUN sudo sh -c 'echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" > /etc/apt/sources.list.d/gazebo-stable.list' && \
+    wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg && \
```

Comment:
nit: Is it necessary the wget? 
I just took a look at: https://gazebosim.org/docs/latest/install_ubuntu/

Reply:
Actually I don't know where I got this way from but I know it worked jaja, I can try out the other way to see if anything changes.

Reply:
I am going to leave it like it is now, because if you see the instructions on the link you passed are the same but using `curl` instead of `wget`, it's not a big change or anything that affects performance.